Website/flaskr/templates/registrazione.html

```diff
@@ -66,36 +66,42 @@ <h1 class="my-5 display-5 fw-bold ls-tight" style="color:black">
                                 </div>
                                 <div class="col-md-6 mb-4">
                                     <div class="form-outline">
-                                        <input type="text" name="telefono"id="telefono" class="form-control" placeholder="Numero di telefono"required/>
+                                        <input type="text" name="telefono" id="telefono" class="form-control" placeholder="Numero di telefono"required/>
                                         <label class="form-label" for="telefono"> </label>
                                     </div>
                                 </div>
                             </div>
 
                             <!-- Email input -->
                             <div class="form-outline mb-4">
-                                <input type="email" name="email"id="email" class="form-control"  placeholder="Email address"required/>
+                                <input type="email" name="email" id="email" class="form-control"  placeholder="Email address"required/>
                                 <label class="form-label" for="email"></label>
                             </div>
 
                             <!-- Password input -->
                             <div class="form-outline mb-4">
-                                <input type="password" name="password"id="password" class="form-control" placeholder="Password"required/>
```

Comment:
Minor fix
in questo file in particolare, molti attribuiti non hanno spazi tra loro, un esempio è: name="email"id="email"

Reply:
Grazie mille, vado a risolvere